client/src/Home.js

```diff
@@ -11,45 +15,47 @@ function Home() {
       </div>
 
       <div className="container">
-        <div className="row">
-          <div className="col-md-3 step">
-            <h3>Step 1</h3>
-            <b>Learn About the Dream Act.</b>
-            <p>
-              The California Dream Act allows undocumented people who went to a california High School for 3 years or more, to get financial
-              aid from the state.
-            </p>
-            <a className="btn btn-primary" href="/">
-              Learn More
-            </a>
+        <Link onClick={() => setActive(false)} to="/resources">
```

Comment:
You've done a good job of styling the cards... but the goal is not to hard-code the categories, but to load them from the database and use code to generate the cards...!